src/includes/performance/enable-tracing/dotnet.mdx

```diff
@@ -0,0 +1,29 @@
+To get started automatically sending traces you can:
```

Comment:
Move this code sample into the `configure-sample-rate` include and delete lines 1-7 as that content is covered in the index file.